src/lib/graphing-rcall.jl

```diff
@@ -31,15 +31,17 @@ function usmap(df, centered=false)
         mapinited = true
     end
 
-    if centered
-        R"ggplot($df) +
+    R"df = $df"
+    R"df$fips = as.numeric(df$fips)"
+    if R"sum(df$value<0)>0 & sum(df$value>0)>0"[1] == 1  
+        R"ggplot(df) +
```

Comment:
I prefer giving the user control on centring.  Want to discuss this?

Reply:
If you prefer, no worries I'll reverse this!

Reply:
Or how about make your operation the default, but allow it go be specified?  Replace `centered=false` with `centered=nothing`, and then on `nothing`, you auto-detect as you have here, and otherwise you follow the user's specification?